runtime/src/ancient_append_vecs.rs

```diff
@@ -81,12 +84,55 @@ impl<'a> AccountsToStore<'a> {
     }
 }
 
+/// capacity of an ancient append vec
+pub fn get_ancient_append_vec_capacity() -> u64 {
+    use crate::append_vec::MAXIMUM_APPEND_VEC_FILE_SIZE;
+    // smaller than max by a bit just in case
+    // some functions add slop on allocation
+    MAXIMUM_APPEND_VEC_FILE_SIZE - 2048
+}
+
+/// true iff storage is ancient size and is almost completely full
+pub fn is_full_ancient(storage: &AppendVec) -> bool {
+    // not sure of slop amount here. Maybe max account size with 10MB data?
+    // append vecs can't usually be made entirely full
+    let threshold_bytes = 10_000;
```

Comment:
The threshold here is 10K even though you mention 10M in the comment. Just confirming this is intended.

Reply:
it is. I'm not sure what the threshold should be. 10k is a guess. 10M might make some sense. But, few accounts are 10MB.